source/features/redirect-to-issue-list.tsx

```diff
@@ -0,0 +1,41 @@
+import * as pageDetect from 'github-url-detection';
+import delegate, {DelegateEvent} from 'delegate-it';
+
+import features from '../feature-manager.js';
+
+function redirectToIssues(event: DelegateEvent<Event, HTMLFormElement>): void {
+	event.preventDefault();
```

Comment:
It doesn't look like it needs to be prevented:

https://stackoverflow.com/a/5640946/288906